### DISCUSSION TITLE
CLI RSS Feed

### DISCUSSION THREAD

**Discussion by bashbunni** [charmbracelet/inspo#8] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would like to see a command line RSS aggregator

**Describe the solution you'd like**
More specifically, I'm looking to use the [Journalist](https://マリウス.com/journalist-an-rss-aggregator/) RSS aggregator on the command line with a TUI. I need to be able to:
- navigate the reader
- navigate the list
- quit the program
- toggle between the list and reader
- refresh feeds
- switch between feeds

**Comment by bashbunni** [comment]
I love https://github.com/mrusme/canard, but it uses tview instead of Bubble Tea. Could be good practice to learn Bubble Tea for someone to re-write it with Bubble Tea! They do use glamour for rendering already

**Comment by mrusme** [comment]
Hey @bashbunni, I just stumbled upon this, happy you dig Journalist and Canard! 👍🏼 

Just wanted to give some feedback, as I'm actually working on bringing Canard to the Bubble Tea ecosystem. Instead of rewriting it as the individual tool that it is right now, however, I'm rather looking for ways to integrate it into [Neon Modem Overdrive](https://github.com/mrusme/neonmodem), since it would offer the perfect foundation for RSS as well. I'm yet undecided for ways to bring interaction to RSS feeds to make them feel like actual an bulletin board, which *Neon Modem* is all about. I'm currently looking at Disqus and [Isso](https://github.com/posativ/isso) to see wheter that would be something worth pursuing.

Long story short, if you have more ideas/suggestions/feedback to share or if there's anyone who's looking to collaborate on this I'd be down for it! 💯

**Reply by bashbunni** [reply]
Amazing! It would be great to collaborate on that :)
